crashtracker-ffi/tests/test_app/src/main.rs

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(not(windows))]
+fn main() {
+    panic!("This test is only supported on Windows");
+}
+
+#[cfg(windows)]
+fn main() {
+    let args: Vec<String> = std::env::args().collect();
+
+    if args.len() < 2 {
+        panic!("Missing argument: crash path");
+    }
+
+    let crash_path = &args[1];
+    println!("Crash path: {}", crash_path);
+
+    // Get the directory of the current exe
+    let exe_path = std::env::current_exe().unwrap();
+    let exe_dir = exe_path.parent().unwrap();
+    let lib_path = exe_dir.join("datadog_crashtracker_ffi.dll");
+
+    init_crashtracking(crash_path, lib_path.to_str().unwrap());
+
+    // Force a segfault to crash
+    let ptr = std::ptr::null_mut::<i32>();
+    unsafe {
+        *ptr = 42;
+    }
```

Comment:
You might want to wrap this in `blackbox` like some of the other tests do to avoid the compiler doing something clever